public/stylesheet.css

```diff
@@ -264,16 +264,27 @@ a {
     margin: 0 0 1rem;
   }
 }
-
 .header__list-item {
-  font-family: var(--font-family-stylized);
-  font-size: 1rem;
-  font-weight: 600;
-  margin: 0 0.7rem;
+  height: 45px;
+  width: 45px;
+  margin: 10px;
+  background-color: var(--color-background);
+  fill: var(--color-button-active);
 }
-.header__list-item:hover {
-  color: var(--color-button-hover);
-  text-decoration: underline;
+
+.header__list-item:hover,
+.header__list-item a:focus {
```

Comment:
This assumes the structure of the document a bit too much I would say, why not just use the `.header__list-icon` selector?

If you think that's not a good solution because styling a class named "icon" in this way is odd, then I'd suggest we introduce a new class for clarity (e.g. `.header__list-link`).